tests/web/APIRest.php

```diff
@@ -400,7 +400,7 @@ public function testUserPicture() {
        */
 
       // Copy pic to tmp folder so it can be set to a user
-      copy("tests/$pic", "files/_tmp/$pic");
+      copy("tests/$pic", "tests/files/_tmp/$pic");
```

Comment:
Maybe you should directly use constants, to be sure that test will not fail if their values change.